crates/wasmtime/src/module.rs

```diff
@@ -1026,7 +1026,10 @@ impl ModuleInner {
     fn memory_images(&self) -> Result<Option<&ModuleMemoryImages>> {
         let images = self
             .memory_images
-            .get_or_try_init(|| memory_images(&self.engine, &self.module))?
+            .get_or_init(|| match memory_images(&self.engine, &self.module) {
+                Ok(v) => v,
+                Err(_e) => None,
+            })
```

Comment:
This no longer returns the error. I did suggest leaving this one alone until get_or_try_init stabilizes.